contracts/margined_pricefeed/src/contract.rs

```diff
@@ -69,8 +69,8 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
         QueryMsg::GetPrice { key } => to_binary(&query_get_price(deps, key)?),
         QueryMsg::GetPreviousPrice {
             key,
-            num_round_back,
-        } => to_binary(&query_get_previous_price(deps, key, num_round_back)?),
+            round_id,
```

Comment:
should remain query structure ?

Reply:
> should remain query structure ?

I have resolved in https://github.com/oraichain/perpetuals/pull/4/commits/d953db3f045c6337df91582244877482d459e140